db/migrate/20160119125950_add_created_on_for_container_entities.rb

```diff
@@ -1,15 +1,44 @@
 class AddCreatedOnForContainerEntities < ActiveRecord::Migration
-  CONTAINER_TABLES = [:container_nodes, :container_projects, :container_services, :container_routes, :container_groups,
-                      :container_replicators, :container_quotas, :container_builds, :container_build_pods,
-                      :container_limits, :container_volumes]
+  class ContainerNode < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  class ContainerProject < ActiveRecord::Base; end
```

Comment:
This was pre-rails 5, but in new migrations created with rails 5, I believe these should be `< ApplicationRecord; end` ... Right @matthewd?